src/sql-parser/src/parser.rs

```diff
@@ -2866,6 +2878,35 @@ impl<'a> Parser<'a> {
         ))
     }
 
+    fn parse_create_webhook_check_options(
+        &mut self,
+    ) -> Result<CreateWebhookSourceValidateOptions<Raw>, ParserError> {
+        let mut secrets = vec![];
+
+        self.parse_comma_separated(|f| match f.expect_one_of_keywords(&[SECRET])? {
+            SECRET => {
+                let secret = f.parse_raw_name()?;
+                let alias = f.parse_optional_alias(|keyword| {
+                    // Note: "Bytes" is reserved in this context since it's used to indicate the
+                    // type we should use to represent this SECRET.
+                    keyword.is_sometimes_reserved() || keyword == Keyword::Bytes
```

Comment:
This prevents `AS bytes BYTES`? I'm ambivalent about this and am also ok allowing it because it makes the parser a bit simpler.

Reply:
Without this, if you tried to specify a secret to have type `BYTES`, the parser would actually interpret that as aliasing the secret to the name "bytes".

I added the following parser test cases:

`SECRET foo bytes` -> a secret named "foo" of type `BYTES`
`SECRET foo AS bytes` -> a secret named "foo" aliased to the name "bytes"
`SECRET foo AS bytes bytes` -> a secret named "foo" aliased to the name "bytes" with type `BYTES`

Reply:
Ah, I was thinking that because we make up the rules for this clause, we should disallow aliases without `AS`. So `SECRET foo AS bar` is allowed, but `SECRET foo bar` is rejected. That avoids the ambiguity, and would also allow us to add additional options in the future (`SECRET foo NO REFRESH`? Idk). But maybe consistency with all the other `AS` clauses is worth it. Not sure!

Reply:
I should have checked this earlier, but it looks like for our other sources we require the `AS` keyword too, I'll make this change!